torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -177,7 +187,9 @@ class TensorExprFuser {
 
     Node* fusion_group = n;
     if (minSubgraphSize_ == 1) {
-      fusion_group = getOrCreateTensorExprSubgraph(n);
+      std::unordered_map<Value*, Value*> vmap;
+      fusion_group = getOrCreateTensorExprSubgraph(n, vmap);
```

Comment:
nitpick: Should we consider making `getOrCreateTensorExprSubgraph` a member method of `TensorExprFuser` so we don't have to thread `vmap`?